### PR TITLE
Feature/be admin module

### PR DESCRIPTION
**Pull request by TheLocagus**
Admin module.
Sending emails.

### CODE REVIEW

**Comment by TheLocagus** [comment]
Enums are to translate to english and DTO to check. I dont know everything about db yet, but anyway it would be good to merge this to develop AFTER merging branch with entities.

File: src/dto/student-to-import.dto.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+export class StudentToImportDto {
+  id?: string;
+  email: string;
+  courseCompletion: number;
+  courseEngagment: number;
+  projectDegree: number;
+  teamProjectDegree: number;
+  bonusProjectUrls: string[];
```
Comment (Adam-DevPL) [inline src/dto/student-to-import.dto.ts]: bonusProjectUrls --> if its dto it can't be string[] 
There is no such a thing in the DB. It can be a string.

**Review by Adam-DevPL — COMMENTED** [review]
I've marked one thing to correct, buy it's repeted more further. 
No such a think as string[] it can be only string in DB so in DTO as well.

File: src/dto/student-to-import.dto.ts (hunk 1 above)
Comment (TheLocagus) [reply src/dto/student-to-import.dto.ts]: Sure! I forgot about it.

**Pull request closed without merging**